client/src/components/App/App.js

```diff
@@ -13,11 +14,13 @@ const App = () => {
       <Router>
         <Switch>
           <Route exact path='/events/:eventProg/:eventCode' component={EventPage} />
-          <Route exact path="/" component={Landing} />
+          <Route exact path='/' component={Landing} />
           <Route exact path="/register/:eventProg/:eventCode/:mobile" 
           component={Register}>
+            <Route exact path='/login' component={AdminLogin}/>
       </Route>
```

Comment:
incorrect Route definition , you nested them in an incorrect way.
```suggestion
          <Route exact path="/register/:eventProg/:eventCode/:mobile"  component={Register} />
            <Route exact path='/login' component={AdminLogin}/>
```